pkg/kubeadm/master/manifests.go

```diff
@@ -128,6 +160,24 @@ func etcdVolumeMount() api.VolumeMount {
 	}
 }
 
+// certsVolume exposes host SSL certificates to pod containers.
+func certsVolume(s *kubeadmapi.KubeadmConfig) api.Volume {
+	return api.Volume{
+		Name: "certs",
+		VolumeSource: api.VolumeSource{
+			// TODO make path configurable
+			HostPath: &api.HostPathVolumeSource{Path: "/etc/ssl/certs"},
+		},
+	}
+}
+
+func certsVolumeMount() api.VolumeMount {
```

Comment:
I think `/var/etcd/certs` is a better location; and we should a) make it configurable (probably bad, one more flag) or b) always use `/var/etcd/certs`, `/var/lib/etcd/certs` or `/var/lib/kubeadm/etcd-certs` whichever we think is better.
`/etc/ssl/certs` seems a bit too generic for me.